picard/util/tags.py

```diff
@@ -165,653 +354,1183 @@ def names(self, selector=None):
+            'Deluxe version with 2 bonus tracks).'
+        ),
+        is_populated_by_picard=False,
+        see_also=('_releasecomment', ),
     ),
     TagVar(
         'compilation',
         shortdesc=N_('Compilation (iTunes)'),
+        longdesc=N_('1 for Various Artist albums, otherwise empty.'),
     ),
     TagVar(
         'composer',
         shortdesc=N_('Composer'),
+        longdesc=N_('The names of the composers for the associated work.'),
+        is_multi_value=True,
     ),
     TagVar(
         'composersort',
         shortdesc=N_('Composer Sort Order'),
+        longdesc=N_('The sort names of the composers for the associated work.'),
+        is_multi_value=True,
     ),
     TagVar(
         'conductor',
         shortdesc=N_('Conductor'),
+        longdesc=N_(
+            'The names of the conductors associated with the track. These can include the conductor '
+            'and chorus master, and could be associated with the release or recording.'
+        ),
+        is_multi_value=True,
     ),
     TagVar(
         'copyright',
         shortdesc=N_('Copyright'),
+        longdesc=N_(
+            'The copyright message for the copyright holder of the original sound, beginning with a '
+            'year and a space character.'
+        ),
+        is_from_mb=False,
     ),
     TagVar(
         'datatrack',
-        shortdesc=N_('FIXME:datatrack'),
+        shortdesc=N_('Data Track'),
+        longdesc=N_('Set to 1 if the track is a "data track", otherwise empty.'),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'date',
         shortdesc=N_('Date'),
+        longdesc=N_('The date that the release (album) was issued, in the format `YYYY-MM-DD`.'),
     ),
     TagVar(
         'director',
         shortdesc=N_('Director'),
+        longdesc=N_(
+            'The director of a track as provided by the "*Video Director*" or "*Audio Director*" relationship '
+            'in MusicBrainz.'
+        ),
+        is_multi_value=True,
     ),
     TagVar(
         'dirname',
         shortdesc=N_('Directory name'),
+        longdesc=N_('The name of the directory containing the file at the point of being loaded into Picard.'),
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'discid',
-        shortdesc=N_('Disc Id'),
+        shortdesc=N_('FreeDB Disc ID'),
+        longdesc=N_('The identification number of the disc in the FreeDB database.'),
+        see_also=('musicbrainz_discid', ),
+        doc_links=(
+            DocumentLink(N_('FreeDB'), 'https://wikipedia.org/wiki/Freedb'),
+            DocumentLink(N_('FreeDB DiscID Calculation'), 'https://wikipedia.org/wiki/CDDB#Example_calculation_of_a_CDDB1_(FreeDB)_disc_ID'),
+            DocumentLink(N_('FreeDB DiscID including Calculation Example (French)'), 'https://fr.wikipedia.org/wiki/DiscId'),
```

Comment:
While it makes total sense to mark the title for translation, it isn't translated in `TagVar.links()`, since `_()` isn't called.

```python
        for doclink in self.doc_links:
            translated_title = _(doclink.title)
            yield f"<a href='{doclink.link}'>{translated_title}</a>"
```

Reply:
Changed in https://github.com/metabrainz/picard/pull/2636/commits/48158eb412dd68c3b7906bedeef8e3be9bfb3ec1